src/main.js

```diff
@@ -2832,7 +2832,18 @@ function addObjBoxResize() {
 }
 
 function patch_m_down() {
-    let og_mdown = window.m_down;
+    // entity.selected fix for when object creation is undone (1/2)
+    let oldCode = m_down.toString();
+    let newCode = oldCode.replace("in newbie.pm) { ldn", `
+        in newbie.pm) {
+        lnd('es[' + newid + '].selected=false;');
+        ldn
+    `);
+    if (oldCode === newCode) { 
+        aleiLog(WARN, "m_down direct code replacement failed (selected fix)");
+    }
+    let og_mdown = eval("(" + newCode + ")");
```

Comment:
Hm? Why do we need `eval` here?

Reply:
Jeje is doing changes on top of function code, and in order to apply changes, you need to get function object, which you can get by evalling.